 Project1/index.html

```diff
@@ -0,0 +1,99 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>
+        Yes, the Octopus Is Smart as Heck. But Why?
+    </title>
+    <link rel="stylesheet" href="style.css">
+</head>
+
+<body>
+
+    <figure>
+        <img src="images/Illustration of Octopus.jpg" alt="desciption of photo" width="800" height="420">
+        <figcaption>
+            An octopus in an aquarium in Brest, France. The animals are highly intelligent, but researchers are
+            uncertain how the trait evolved
+            Fred Tanneau/Agence France-Presse — Getty Images
+        </figcaption>
+    </figure>
+    <h1>Yes, the Octopus Is Smart as Heck. But Why?
+    </h1>
+    <h2> It has eight arms, three hearts — and a plan. Scientists aren’t sure how the cephalopods got to be
+        so intelligent.</h2>
+    <p>
+        <h2>
+            <img src="Carl Zimmer.png" alt="photo of the author" width="200" height="200">
```

Comment:
Same note here about the spaces in the filename and `alt` text, plus the `src` is missing the directory name, so the image is not showing up in the browser:
```suggestion
            <img src="images/CarlZimmer.png" alt="photo of the author" width="200" height="200">
```